src/Native/Unix/System.Native/pal_networkstatistics.c

```diff
@@ -482,6 +488,18 @@ int32_t SystemNative_GetNativeIPInterfaceStatistics(char* interfaceName, NativeI
             retStats->OutMulticastPackets = systemStats.ifi_omcasts;
             retStats->InDrops = systemStats.ifi_iqdrops;
             retStats->InNoProto = systemStats.ifi_noproto;
+
+            retStats->Flags = 0;
+            if (ifHdr->ifm_flags & IFF_UP)
+            {
+                retStats->Flags |= InteropInterfaceUp;
+            }
+
+            if (ifHdr->ifm_flags & (IFF_MULTICAST | IFF_ALLMULTI))
```

Comment:
This is currently stating that multicast is supported if either the IFF_MULTICAST or the IFF_ALLMULTI flag is set... it's correct that's it's either, rather than, say, requiring both?

Reply:
correct. When IFF_MULTICAST is set you can receive traffic for subscribed multicast addresses. 
When IFF_ALLMULTI you can receive traffic even for groups you did not subscribed to. Either one is sign that multicast operations are supported. When neither flag is set, setting subscription would probably fail. (at least is did on Linux)